backend/src/main/java/com/github/EazyErik/service/DrinksService.java

```diff
@@ -40,7 +41,7 @@ public List<Drink> getAllMyFavourites(String username) {
     public void removeFromFavs(String id) {
 
 
-        drinksRepository.deleteById(id);
+        favouriteDrinksRepository.deleteById(id);
```

Comment:
Maybe check if the deleted drink is a fav of the current user, otherwise every user could delete favs of any user